wpilib/wpilib/lineardigitalfilter.py

```diff
@@ -76,15 +80,12 @@ class LinearDigitalFilter(Filter):
     
     """
     
-    def __init__(self, source, ffGains, fbGains):
+    def __init__(self, source: PIDSource, ffGains: Union[List[float], Tuple[float]], fbGains: Union[List[float], Tuple[float]]) -> None:
```

Comment:
The type hint for `source` still doesn't match the original documented type.

Also, there's a much saner way of specifying the types for `ffGains` and `fbGains` which would make much more sense in this scenario.

Reply:
I bet it has to do with 'Iterable'

Reply:
You want a `Sequence` here, because `Iterable` won't type check.